.github/workflows/build-and-test.yml

```diff
@@ -77,6 +77,12 @@ jobs:
           sed -i -e "/pandas/d" -e "/pyarrow/d" python/requirements-dev.txt
           conda install -c conda-forge --yes --file python/requirements-dev.txt
           conda list
+          echo PRANAVSUKUMAR
+          pwd
+          cd python
```

Comment:
question, will these folder like `python`, `delta-kernel-python`, always exist?

Reply:
in these automation tests, yes (as long as we don't change the names or structures of these folders")